### ISSUE TITLE
Add new rule for detecting CSS comments

### ISSUE THREAD

**Issue by scripthunter7** [AdguardTeam/AGLint#235]
e.g.
```adblock
#$#body { padding: 0; /* foo */ }
##.ad /* bar */
```

Suggested rule name: `no-css-comments`

Can be autofixed

Note:

By default, AGTree is tolerant and will parse these rules. However, we should throw an error for such rules.

Additionally, the extension will discard them too.
